packages/config/config/devices/0x0384/ha-zw-5sab.json

```
@@ -137,47 +131,48 @@
 		},
 		{
 			"#": "111",
-			"label": "Configuration Temperature Change Threshold",
-			"description": "The temperature change threshold value(unit 0.1℃)",
+			"label": "Temperature Change Threshold",
+			"description": "Units are fraction of a full ℃ (0.1℃ increments)",
 			"valueSize": 2,
+			"unit": "°C",
 			"minValue": 0,
 			"maxValue": 10,
 			"defaultValue": 10,
 			"unsigned": true
 		},
 		{
 			"#": "112",
-			"label": "Configuration Humidity Change Threshold",
-			"description": "Humidity change threshold value(unit %)",
+			"label": "Humidity Change Threshold",
 			"valueSize": 1,
+			"unit": "%",
 			"minValue": 0,
 			"maxValue": 5,
 			"defaultValue": 5,
 			"unsigned": true
 		},
 		{
 			"#": "113",
-			"label": "Configuration Luminance Change Threshold",
-			"description": "Configuration luminance change threshold. valid values: 1-65528(unit lx)",
+			"label": "Luminance Change Threshold",
 			"valueSize": 2,
-			"minValue": -32767,
-			"maxValue": 32766,
+			"unit": "Lux",
+			"minValue": 1,
+			"maxValue": 65528,
 			"defaultValue": 150,
 			"unsigned": true
 		},
 		{
 			"#": "114",
-			"label": "Configuration Battery Level Change Threshold",
-			"description": "Battery level change threshold value(unit)%",
+			"label": "Battery Level Change Threshold",
 			"valueSize": 1,
+			"unit": "%",
 			"minValue": 0,
 			"maxValue": 10,
 			"defaultValue": 10,
 			"unsigned": true
 		},
 		{
 			"#": "12",
-			"label": "Turn On/Off Pir",
+			"label": "Pir Sensitivity",
 			"description": "0x0a indicates the highest sensitivity, 0x00 means off PIR detection function",
```

```suggestion
			"description": "1 = lowest sensitivity, 10 = highest sensitivity",
```
and 0 should be a single option "Disable"